### DISCUSSION TITLE
How can we add tooltip for multi-line charts.

### DISCUSSION THREAD

**Discussion by vijaydiyyala842** [Abhinandan-Kushwaha/react-native-gifted-charts#909] [Q&A]
I add screenshot for UX reference below.
![Screenshot 2024-11-18 at 10 58 02 AM](https://github.com/user-attachments/assets/e0a44e59-b7a5-40a4-a81a-6e678b37e17a) 

Please give some inputs for this. As I am unable figure it out. My code as follows below.

<View style={{ flex: 1 }}>
              <LineChart
                height={300}
                width={300}
                dashGap={0}
                yAxisLabelWidth={40}
                yAxisLabelContainerStyle={{ paddingRight: 5 }}
                yAxisTextStyle={styles.yaxisText}
                xAxisLabelTextStyle={styles.yaxisText}
                xAxisLength={Dimensions.get("screen").width - 80}
                rulesLength={Dimensions.get("screen").width - 80}
                rulesColor={"#E7E7E7"}
                isAnimated
                hideDataPoints
                focusEnabled
                showDataPointLabelOnFocus
                spacing={36}
                noOfSections={6}
                dataSet={dataset}
                yAxisThickness={0}
                xAxisColor="#E7E7E7"
              />
            </View>

**Comment by Abhinandan-Kushwaha** [comment]
Hi @vijaydiyyala842 
I have answered this here- https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/issues/910#issuecomment-2562012436
